src/components/AddressVerificationContainer/AddressVerificationContainer.module.css

```diff
@@ -1,7 +1,11 @@
 .multipleAddressList {
   display: flex;
-  padding: 0;
-  list-style: none;
+}
+
+@media only screen and (min-width: 400px) {
+  .checkedClass {
+    margin-top: 0 !important;
```

Comment:
Hey, could you walk me through why we’re using !important here? We usually try to avoid it because it can override other styles in a way that makes the CSS harder to manage and maintain later on.

Reply:
CSS applies styles based on selector specificity, and the one with higher specificity will take precedence. You can try to add another selector for that and give it a try  e.g. `.some-wrapper-container-class .checkedClass`

Reply:
No need for a wrapper, we can use the existing selector in the DOM tree.

For example, if we use the `id`, this should work.

```
 #address-verification-container li {
  margin-top: 0;
}
```

Reply:
Perhaps it just needs a rebuild on the project locally? This code seem to fix the bug when I run it locally.